### PR TITLE
add a new icon button

### PR DESCRIPTION
**Pull request by alkatrivedi**
Issue: #1

### CODE REVIEW

**Comment by alkatrivedi** [comment]
@shahednasser actually I was not getting how to add that add-to-cart icon in the buttons. So for now I just created a button of add to cart, can you please guide me so that I can made desired changes. Thank you.

**Comment by shahednasser** [comment]
Great work @alkatrivedi  thank you! I just have the following requests:

1. Can you pull the changes from this repository, as some changes are conflicting with previous work?
2. Place the "add to cart" button in the correct alphabetical order in `assets/js/buttons-examples.js` (basically it should be first under icon buttons).
3. Make sure to run `npm run compile` to compile the less files into `dist/sbuttons.min.css`. This is how you can see the changes you make.

If you need more help please ask. You can also check out the [contributing guide](https://github.com/shahednasser/sbuttons/blob/master/CONTRIBUTING.md) for more assistance.

**Comment by alkatrivedi** [comment]
Done that. Can you please check it. If there's anything wrong I did please let me know. Thank you for your help:)

**Comment by shahednasser** [comment]
Ok @alkatrivedi I made some changes to make sure the changes from this repo are not overwritten. It looks good now. You just need to run `npm run compile` and commit the changes to see how the button looks like on the website.

**Comment by alkatrivedi** [comment]
Actually working tree is clean; there is nothing to commit.

**Comment by shahednasser** [comment]
Running `npm run compile` makes changes to `dist/sbuttons.min.css` and `dist/sbuttons.css`, but I'm not seeing any change in these files. Are you sure you ran it? If so, can you show me what the output was?  
Make sure you ran `npm install` first to install the dependencies.

**Comment by alkatrivedi** [comment]
Did that. I think dependencies was not installed that is why that was happening, now I am able to see the changes but the only thing is add to cart button has same icon as that of buy me a coffee that's what I was not getting how to change that icon.

File: dist/sbuttons.css [hunk 1]
```diff
@@ -732,6 +732,24 @@ a.yellow-btn:active:not(.fill-color-btn) {
   height: 1.6em;
   padding: 0;
 }
+.add-to-cart-btn {
+  position: relative;
+  padding-right: 1.5em;
+}
+.add-to-cart-btn:after {
+  content: "\f7b6";
```
Comment (shahednasser) [inline dist/sbuttons.css:740]: The icon of the button is a mug which I think is a mistake, please change the code to the icon of a shopping cart which is "\f07a"

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: dist/sbuttons.css (hunk 1 above)
Comment (alkatrivedi) [reply dist/sbuttons.css:740]: I am changing it but after running npm run compile it is automatically changing to "\f7b6".

Comment (shahednasser) [reply dist/sbuttons.css:740]: Oh sorry, I meant to change it in the less file not in the css. Please do that then compile.

Comment (alkatrivedi) [reply dist/sbuttons.css:740]: Okay

Comment (alkatrivedi) [reply dist/sbuttons.css:740]: I don't know why I am getting this error while pushing.

Comment (alkatrivedi) [reply dist/sbuttons.css:740]: I google this error and it says I need to do force pushing should I?

Comment (alkatrivedi) [reply dist/sbuttons.css]: Done that. Thank You!
Now can you please check it.

Comment (alkatrivedi) [reply dist/sbuttons.css]: Can you please check it. I did that and I can see the desired changes.

**Comment by shahednasser** [comment]
Awesome thank you!

@all-contributors please add @alkatrivedi for code

**Comment by alkatrivedi** [comment]
Thank you.

**Pull request merged**